wolfcrypt/src/evp.c

```diff
@@ -593,6 +613,299 @@ WOLFSSL_API int wolfSSL_EVP_add_digest(const WOLFSSL_EVP_MD *digest)
     return 0;
 }
 
+
+WOLFSSL_API int wolfSSL_EVP_PKEY_CTX_free(WOLFSSL_EVP_PKEY_CTX *ctx)
```

Comment:
This function should have a comment about the return codes, why 1 instead of WOLFSSL_SUCCESS?  As well as the next several functions.